packages/rancher-prom2teams/generated-changes/patch/templates/NOTES.txt.patch

```diff
@@ -0,0 +1,8 @@
+--- charts-original/templates/NOTES.txt
++++ charts/templates/NOTES.txt
+@@ -1,2 +1,3 @@
+-Prom2Teams has been installed. Check its status by running:
+-  kubectl --namespace {{ .Release.Namespace }} get pods -l "app.kubernetes.io/instance={{ .Release.Name }}"
++rancher-prom2teams is now installed on the cluster!
++Please refer to the upstream documentation for configuration options:
++https://github.com/idealista/prom2teams
```

Comment:
nit: since this will never be seen, we could just drop this patch file entirely and leave it as default

Reply:
fixed.